e2e/specs/clinical-forms.spec.ts

```diff
@@ -43,19 +43,31 @@ test('Fill a clinical form', async ({ page, api }) => {
     await expect(chartPage.page.getByRole('cell', { name: /surgical operation/i })).toBeVisible();
   });
 
-  await test.step('And if I fill a form', async () => {
+  await test.step('When I click the `Soap note template` link to launch the form', async () => {
     await chartPage.page.getByText(/soap note template/i).click();
+  });
 
-    await expect(chartPage.page.getByRole('button', { name: /save and close/i })).toBeVisible();
-    await expect(chartPage.page.getByRole('button', { name: /discard/i })).toBeVisible();
+  await test.step('Then I should be redirected to `Soap note template` form', async () => {
```

Comment:
Same here. Something like: `Then I should see the 'Soap Note template' form in the workspace` sounds preferable to me.